src/when-you-get-an-alert.html

```diff
@@ -63,10 +66,13 @@ <h1 class="govuk-heading-xl">
         <p>The government and mobile phone networks are <a href="/alerts/planned-tests">testing emergency alerts</a>. You may get an alert if you live in, or travel through, a test area.</p>
       </div>
       <h2 class="govuk-heading-m">
-        If you’re driving when you get an alert
+        If you’re driving or riding when you get an alert
```

Comment:
```suggestion
        If you’re travelling when you get an alert
```

We've set "travel" above, so can we say it here?

Reply:
Travelling is not the same as driving or riding.

If you’re travelling – on a train or as a passenger in a motor vehicle – there’s nothing to stop you checking your device.

The same is not true for anyone responsible for driving/riding a vehicle.